components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -241,6 +257,43 @@ def generate_container_config(clp_config: CLPConfig, clp_home: pathlib.Path):
     return container_clp_config, docker_mounts
 
 
+def dump_container_config(
+    clp_config: CLPConfig, container_clp_config: CLPConfig, container_name: str
+) -> Tuple[pathlib.Path, pathlib.Path]:
```

Comment:
```suggestion
) -> Tuple[pathlib.Path, pathlib.Path]:
    """
    Writes the given config to the logs directory so that it's accessible in the container.
    :param clp_config: The corresponding config on the host (used to determine the logs directory).
    :param container_clp_config: The config to write.
    :param container_name:
    :return: The path to the config file in the container and on the host.
    """
```